lib/core/src/base64.cpp

```diff
@@ -4,135 +4,139 @@
 
 #include "irods/rodsClient.h"
 #include "irods/base64.h"
+
 #define OK 0
```

Comment:
Consider removing this and replacing all usage with `0`.

Reply:
done